api/diagnostics.go

```diff
@@ -840,18 +843,18 @@ func matchRequestedNodes(requestedNodes []string, masterNodes, agentNodes []dcos
 	if len(matchedNodes) > 0 {
 		return matchedNodes, nil
 	}
-	return matchedNodes, fmt.Errorf("Requested nodes: %s not found", requestedNodes)
+	return matchedNodes, fmt.Errorf("requested nodes: %s not found", requestedNodes)
 }
 
 func findRequestedNodes(requestedNodes []string, tools dcos.Tooler) ([]dcos.Node, error) {
 	masterNodes, err := tools.GetMasterNodes()
 	if err != nil {
-		logrus.WithError(err).Errorf("Could not get master nodes")
+		return nil, fmt.Errorf("could not get master nodes: %s", err)
```

Comment:
Should we use `%w` here?

Reply:
This is error that will be parsed into JSON. We have no logic around it so we will not use any features of wrapping errors.